erpnext/manufacturing/doctype/work_order/work_order.py

```diff
@@ -470,17 +470,35 @@ def set_work_order_operations(self):
 		"""Fetch operations from BOM and set in 'Work Order'"""
 
 		def _get_operations(bom_no, qty=1):
-			return frappe.db.sql(
+			data = frappe.db.sql(
 					f"""select
 						operation, description, workstation, idx,
 						base_hour_rate as hour_rate, time_in_mins * {qty} as time_in_mins,
-						"Pending" as status, parent as bom, batch_size, sequence_id
+						"Pending" as status, parent as bom, batch_size, sequence_id, fixed_time
 					from
 						`tabBOM Operation`
 					where
-						parent = %s order by idx
+						parent = %s
+						and fixed_time = 0
+						order by idx
 					""", bom_no, as_dict=1)
 
+			for o in frappe.db.sql(
```

Comment:
This doesn't seem to be correct. Why the same query is running twice and added to output?

Reply:
It's not the same query, on time with "and fixed_time = 0" and the other time with and fixed_time = 1

Reply:
I can tell you it a needed feature not just for me but because I work with other ERP that manage production process (open source and proprietary) and it a feature they've got so ERPNext, to my humble point of view need it.As you can see another user ask for it and I can swear I don'tt know him, I was even surprise to see a comment on my PR. 
Would you like me to ask on forum to see what user tell ?